web/template/certificate_provider_your_details.gohtml

```diff
@@ -1,27 +1,19 @@
 {{ template "page" . }}
 
 {{ define "serviceName" }}{{ tr .App "beACertificateProvider" }}{{ end }}
-{{ define "pageTitle" }}{{ tr .App "yourDetails" }}{{ end }}
+{{ define "pageTitle" }}{{ tr .App "enterYourDetails" }}{{ end }}
 
 {{ define "main" }}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <form novalidate method="post">
-        <h1 class="govuk-heading-xl">{{ tr .App "yourDetails" }}</h1>
-
-        <h2 class="govuk-heading-m">{{ tr .App "donor" }}</h2>
-        <p class="govuk-body">{{ .Lpa.Donor.FullName }}</p>
-
-        <h2 class="govuk-heading-m">{{ tr .App "yourDetails" }}</h2>
-        <p class="govuk-body">{{ .Lpa.CertificateProvider.FullName }}</p>
+        <h1 class="govuk-heading-xl">{{ tr .App "enterYourDetails" }}</h1>
 
         {{ template "date" (input . "date-of-birth" "dateOfBirth" .Form.Dob "hint" "dateOfBirthHint") }}
 
-        {{ template "dob-warning" . }}
-
-        {{ template "input" (input . "email" "email" .Form.Email "classes" "govuk-input--width-20" "type" "email" "spellcheck" "false" "autocomplete" "email") }}
+        {{ template "warning" (warning .App "18OrOverWarning") }}
 
-        {{ template "input" (input . "mobile" "mobile" .Form.Mobile "classes" "govuk-input--width-20 govuk-!-margin-bottom-6" "type" "tel" "autocomplete" "tel") }}
+        {{ template "input" (input . "mobile" "Enter your UK mobile number" .Form.Mobile "context" "enterYourUkMobileContext" "hint" "enterYourUkMobileHint" "classes" "govuk-input--width-20 govuk-!-margin-bottom-6" "type" "tel" "autocomplete" "tel") }}
```

Comment:
Also hate this being "Enter your UK mobile number" and not "Your UK mobile number", we don't say "Enter X" for anything else. And the context text will make no sense if they've already signed and witnessed the LPA.

Reply:
Anyway, I think you'll want "Enter your UK mobile number"  to be translated

Reply:
Ah yeah missed this translation when I was getting the markup right, will switch out for a translation key.

And yes agree - we'll need to vary the context based on LPA status here. I'll flag for some updated wording.